src/common/platform_info.ts

```diff
@@ -24,14 +25,15 @@ const platformInfo = {
   isSnap: p.env.ELECTRON_SNAP,
   isDevelopment: process.env.NODE_ENV !== 'production',
   isAppImage: p.env.DESKTOPINTEGRATION === 'AppImageLauncher',
+  isDevEnv: isDevEnv,
```

Comment:
You should update `isDevelopment` here instead of making a new key. Checking `isPackaged` is better than checking the NODE_ENV